# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/translator/AtomicCounterFunctionHLSL.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/compiler/translator/AtomicCounterFunctionHLSL.h
--- a/gfx/angle/checkout/src/compiler/translator/AtomicCounterFunctionHLSL.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/compiler/translator/AtomicCounterFunctionHLSL.h	Sat Sep 07 13:38:36 2019 +0000
@@ -24,6 +24,8 @@
 class AtomicCounterFunctionHLSL final : angle::NonCopyable
 {
   public:
+    AtomicCounterFunctionHLSL(bool forceResolution);
+
     ImmutableString useAtomicCounterFunction(const ImmutableString &name);
 
     void atomicCounterFunctionHeader(TInfoSinkBase &out);
@@ -38,6 +40,7 @@
     };
 
     std::map<ImmutableString, AtomicCounterFunction> mAtomicCounterFunctions;
+    bool mForceResolution;
 };
 
 ImmutableString getAtomicCounterNameForBinding(int binding);